cristin-project/src/main/java/no/unit/nva/cristin/projects/ProjectStatus.java

```diff
@@ -7,28 +7,52 @@
 import static java.util.stream.Collectors.joining;
 
 public enum ProjectStatus {
-    NOTSTARTED,
-    ACTIVE,
-    CONCLUDED;
+    NOTSTARTED("not started"),
+    ACTIVE("active"),
+    CONCLUDED("concluded");
 
     public static final String ERROR_MESSAGE_TEMPLATE = "%s not a valid ProjectStatus, expected one of: %s";
     public static final String DELIMITER = ", ";
 
+    private final String cristinStatus;
+
+    ProjectStatus(String cristinStatus) {
+        this.cristinStatus = cristinStatus;
+    }
+
     /**
-     * Lookup project status by string value.
+     * Lookup ProjectStatus by string from cristin value.
      *
      * @param name assumed status name
      * @return enum value associated with name
      */
-    public static ProjectStatus lookup(String name) {
+    public static ProjectStatus fromCristinStatus(String name) {
         return stream(values())
-                .filter(nameType -> nameType.name().equalsIgnoreCase(name))
+                .filter(nameType -> nameType.getCristinStatus().equalsIgnoreCase(name))
                 .findAny()
                 .orElseThrow(() -> returnException(name));
     }
 
     public static boolean isValidStatus(String nameCandidate) {
-        return stream(values()).anyMatch(enumName -> enumName.name().equalsIgnoreCase(nameCandidate));
+        return stream(values()).anyMatch(enumName -> isValidAnyStatus(nameCandidate, enumName));
+    }
+
+    /**
+     * Maps to a ProjectStatus from given string.
+     *
+     * @param name string to map
+     * @return corresponding ProjectStatus in NVA
+     */
+    public static ProjectStatus getNvaStatus(String name) {
+        return stream(values())
+                .filter(nameType -> nameType.name().equalsIgnoreCase(name))
+                .findFirst()
+                .get();
```

Comment:
I'd be tempted by a Commons::SingletonCollector that throws if the input is not a member